pkg/pillar/types/attesttypes.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2020 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package types
+
+//AttestNonce carries nonce published by requester
+type AttestNonce struct {
+	nonce     []byte
+	requester string
+}
+
+//Key returns nonce content, which is the key as well
```

Comment:
Do you always unpublish the old ones? If not you end up with a bunch of files in /var/run/zedmanager/AttestNonce

I assume there would only be one outstanding nonce at a time, and only one requester?
The key for this could be the requester?

Reply:
> Do you always unpublish the old ones? If not you end up with a bunch of files in /var/run/zedmanager/AttestNonce

Yes, we need to unpublish old one before publishing a new one, as there can be only one outstanding nonce request from a requester (client/zedagent)

> I assume there would only be one outstanding nonce at a time, and only one requester?
> The key for this could be the requester?

Changed the key to requester string.

Reply:
OK